post-update

```diff
@@ -26,8 +26,8 @@ fi
 
 # Update the backend
 pip install -r requirements-pip.txt
-pip install -r requirements-prequ.txt
-prequ sync requirements.txt
+# pip install -r requirements-prequ.txt
+# prequ sync requirements.txt
```

Comment:
Why is prequ commented out?

Reply:
Prequ is an unmaintained fork from pip-tools and it does not compile with the upgraded environment thus, prequ sync is now replaced in the following [commit](https://github.com/City-of-Turku/parkkihubi/pull/6/commits/f76dc54cdf727d39445861bb6de6c8e69d662d03) with pip install.

Reply:
Ok, then it was a good decision to comment it out :) Why just not remove it?

Reply:
This was my mishap, I did the out commenting whilst debugging and forgot to return to it once things workout.
It was initially supposed to be replaced, not only be commented out. 

Now in the commit I linked above, you can see that the prequ lines are gone and instead there is `pip install` line to handle the requirements.

Reply:
Good!